SocialCareCaseViewerApi/V1/Boundary/Requests/ListAllocationsRequest.cs

```diff
@@ -16,6 +16,9 @@ public class ListAllocationsRequest
 
         [FromQuery(Name = "worker_email")]
         public string WorkerEmail { get; set; }
+
+        [FromQuery(Name = "team_id")]
+        public long TeamId { get; set; }
```

Comment:
**nitpick (non-blocking)**: For all the long properties in this class it might be tidier if we made them nullable. I don't think the current setup has any bugs though as teamid, workerid and mosaicid are never 0 in the DB.

My opinion is that it is clearer if we say a property has a value of null if nothing has been supplied instead of assuming the default value of 0 means the API did not receive a value for this property.

Feel free to ignore because effectively the code will work the same. If you do make the properties nullable instead of == 0 checks or != 0 they would have to become == null or != null. I think this applies in validation logic and usecase.